google/docs/list_docs.py

```diff
@@ -0,0 +1,36 @@
+from googleapiclient.errors import HttpError
+
+from auth import client
+
+
+def main():
+    service = client('drive', 'v3')
+    try:
+        # Query Google Drive for all documents
+        query = "mimeType='application/vnd.google-apps.document'"
+        page_token = None
+        while True:
+            results = service.files().list(q=query,
+                                           pageSize=10,
+                                           fields="nextPageToken, files(id, name)",
+                                           pageToken=page_token,
+                                           ).execute()
+
+            items = results.get('files', [])
+            if not items:
+                print('No documents found.')
+                break
+
+            for item in items:
+                print(f"{item['name']} (ID: {item['id']})")
```

Comment:
I'll follow up with support for datasets after I get the other tools I'm working on functional.